apps/meteor/tests/data/livechat/rooms.ts

```diff
@@ -208,6 +209,21 @@ export const sendMessage = (roomId: string, message: string, visitorToken: strin
 	});
 };
 
+export const uploadFile = (roomId: string, visitorToken: string): Promise<IMessage> => {
+	return new Promise((resolve, reject) => {
+		request
+			.post(api(`livechat/upload/${roomId}`))
+			.set({ 'content-type': '', 'x-visitor-token': visitorToken, ...credentials })
+			.attach('file', imgURL)
+			.end((err: Error, res: DummyResponse<IMessage>) => {
+				if (err) {
+					return reject(err);
+				}
+				resolve(res.body as unknown as IMessage);
```

Comment:
Meteor methods return a non error response even for errors. Better check if it actually worked before converting the type

Reply:
Saw you added an if to another function but not to this one 🤔